packages/odyssey-react/src/components/Button/Button.module.scss

```diff
@@ -15,38 +15,38 @@
 
   display: inline-block;
   position: relative;
-  margin-block: var(--SpaceMarginBlock);
-  margin-inline: var(--SpaceMarginInline);
-  padding-block: var(--SpacePaddingBlock);
-  padding-inline: var(--SpacePaddingInline);
+  margin-block: var(--MarginBlock);
+  margin-inline: var(--MarginInline);
+  padding-block: var(--PaddingBlock);
+  padding-inline: var(--PaddingInline);
   transition-property: color, background-color, border-color, box-shadow;
   transition-duration: var(--TransitionDuration);
   transition-timing-function: var(--TransitionTiming);
   border-width: var(--BorderWidth);
   border-style: var(--BorderStyle);
-  border-color: var(--ColorBorderPrimary);
-  background-color: var(--ColorBackgroundPrimary);
-  color: var(--ColorText);
+  border-color: var(--PrimaryBorderColor);
+  background-color: var(--PrimaryBackgroundColor);
+  color: var(--TextColor);
   font-family: var(--FontFamily);
   font-size: var(--FontSize);
   font-weight: var(--FontWeight);
-  line-height: var(--FontLineHeight);
-  white-space: var(--FontWhiteSpace);
+  line-height: var(--LineHeight);
+  white-space: var(--WhiteSpace);
 
   &:hover {
-    border-color: var(--ColorBorderPrimaryHover);
-    background-color: var(--ColorBackgroundPrimaryHover);
+    border-color: var(--PrimaryHoverBorderColor);
+    background-color: var(--PrimaryHoverBackgroundColor);
   }
 
   &:focus {
     @include outline;
```

Comment:
consider pulling out the styles here and removing this mixin so they can be themed - we need to allow border styles and colors to be updated here for focus styles on button!

Reply:
Will update!